consensus/misc/kip71.go

```diff
@@ -42,17 +42,24 @@ func NextBlockBaseFee(parentHeader *types.Header, config *params.ChainConfig) *b
 	gasTarget := config.Governance.KIP71.GasTarget
 	upperGasLimit := config.Governance.KIP71.MaxBlockGasUsedForBaseFee
 
+	// check the case of upper/lowerBoundBaseFee is updated by governance mechanism
 	parentBaseFee := parentHeader.BaseFee
+	if parentBaseFee.Cmp(upperBoundBaseFee) >= 0 {
+		parentBaseFee = upperBoundBaseFee
+	} else if parentBaseFee.Cmp(lowerBoundBaseFee) <= 0 {
+		parentBaseFee = lowerBoundBaseFee
+	}
+
 	parentGasUsed := parentHeader.GasUsed
 	// upper gas limit cut off the impulse of used gas to upper bound
 	if parentGasUsed > upperGasLimit {
 		parentGasUsed = upperGasLimit
 	}
 	if parentGasUsed == gasTarget {
-		return new(big.Int).Set(parentHeader.BaseFee)
+		return new(big.Int).Set(parentBaseFee)
 	} else if parentGasUsed > gasTarget {
 		// shortcut. If parentBaseFee is already reached upperbound, do not calculate.
-		if parentBaseFee.Cmp(upperBoundBaseFee) >= 0 {
+		if parentBaseFee.Cmp(upperBoundBaseFee) == 0 {
```

Comment:
Is it okay? https://github.com/klaytn/klaytn/pull/1477#discussion_r912551052 refers `>=` is right, not `==`

Reply:
I think this can be changed to `==` after applying L47-L51 changes.

Reply:
I got it, Thansk @jimni1222